src/app.js

```diff
@@ -222,15 +237,35 @@ class SAFEApp extends EventEmitter {
           file = await emulation.fetch(filePath);
         }
         const openedFile = await emulation.open(file, consts.pubConsts.NFS_FILE_MODE_READ);
-        const data = await openedFile.read(
-          consts.pubConsts.NFS_FILE_START, consts.pubConsts.NFS_FILE_END);
+        const fileSize = await openedFile.size() || '*';
+        let data;
+
+        // TODO: how do we handle multipart Reqs
+        if (options && options.range) {
```

Comment:
We can remove the conditionals here and define start & end on L152 for both cases and then here keep it as:
```
data = await openedFile.read(start, end);
```